frontend/shared-components/external-link/external-link.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './external-link.module.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons'
+
+interface ExternalLinkProps { linkPath: string, linkText: string }
+
+export default function ExternalLink ({ linkPath, linkText }: ExternalLinkProps) {
+  const {externalLink} = styles
+  return (
+    <a 
+      className={externalLink} 
+      href={linkPath}
```

Comment:
Oh awesome! Yeah so I think the screen reader warning just needs to be an aria-label so that screen reader will read linkText + "external" or something like that

[WebAIM on external links](https://webaim.org/techniques/hypertext/hypertext_links#new_window)